dev/NavigationView/NavigationView.cpp

```diff
@@ -3789,6 +3791,17 @@ void NavigationView::OnPropertyChanged(const winrt::DependencyPropertyChangedEve
     else if (property == s_AutoSuggestBoxProperty)
     {
         InvalidateTopNavPrimaryLayout();
+        if (args.OldValue())
+        {
+            m_autoSuggestBoxQuerySubmittedRevoker.revoke();
+        }
+        if (const auto newSuggestBox = args.NewValue())
+        {
+            if (const auto autoSuggestBox = newSuggestBox.try_as<winrt::AutoSuggestBox>())
```

Comment:
why not `args.NewValue().try_as<winrt::AutoSuggestBox>()`?

Reply:
What if args.NewValue() is nullptr?

Reply:
try_as should cover that case

Reply:
Yep you are right, updated that.